cloudmetrics/metrics/woi.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import numpy as np
+import matplotlib.pyplot as plt
+import pywt
+
+def _debug_plot(cloud_scalar, k, specs):
+    labs = ["Horizontal", "Vertical", "Diagonal"]
+    fig, axs = plt.subplots(ncols=2, figsize=(8, 4))
+    axs[0].imshow(cloud_scalar, "gist_ncar")
+    axs[0].set_xticks([])
+    axs[0].set_yticks([])
+    axs[0].set_title("CWP")
+    for i in range(3):
+        axs[1].plot(k[1:], specs[:, i], label=labs[i])
+    axs[1].set_xscale("log")
+    axs[1].set_xlabel(r"Scale number $k$")
+    axs[1].set_ylabel("Energy")
+    axs[1].set_title("Wavelet energy spectrum")
+    axs[1].legend()
+    plt.tight_layout()
+    plt.show()
+
+
+def woi(cloud_scalar, pad_method='periodic', wavelet='haar', separation_scale=5, debug=False, return_spectra=False):
+    """
+    Computes the three Wavelet Organisation Indices WOI1, WOI2, WOI3 proposed by 
+    Brune et al. (2018) from the stationary/undecimated Direct Wavelet Transform
+    of a scalar field.
```

Comment:
Could you include the DOI or URL to the paper (https://doi.org/10.1002/qj.3409)?